ucm2/sof-soundwire/rt713-sdca.conf

```diff
@@ -4,17 +4,25 @@ SectionDevice."Headphones" {
 	Comment	"Headphones"
 
 	EnableSequence [
+		cset "name='rt713 FU05 Playback Switch' on"
 		cset "name='Headphone Switch' on"
 	]
 
 	DisableSequence [
 		cset "name='Headphone Switch' off"
+		cset "name='rt713 FU05 Playback Switch' off"
+	]
+
+	ConflictingDevice [
```

Comment:
I don't like this settings. Is there other reason other than LED control to mark those devices conflicting together?

Reply:
@perexg The customer's machine has one speaker mute LED and also wants to mute the output of the headphones.
Therefore, we added device conflicting to avoid the speaker being on while the headphones are plugged.
e.g.
We registered two controls for the speaker mute LED.
attach:rt1318-1 DAC Switch
attach:rt713 FU05 Playback Switch
The kernel driver (control_led.c) will check these two controls. These two controls turn off, LED could turn ON.
Therefore, we set conflicting devices for the speaker and headphones each other.